web/static/actor/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Actor Info</title>
+    <script type="module" src="/ux/curio-ux.mjs"></script>
+    <script type="module" src="/actor-summary.mjs"></script>
```

Comment:
We don't need /actor-summary here

Reply:
That's where <sector-expirations> live, which we use below